lib/vnet/daemon/service_darwin.go

```diff
@@ -77,6 +78,7 @@ func Start(ctx context.Context, workFn func(context.Context, Config) error) erro
 		"ipv6_prefix", config.IPv6Prefix,
 		"dns_addr", config.DNSAddr,
 		"home_path", config.HomePath,
+		"cred", fmt.Sprintf("%#v", config.ClientCred),
```

Comment:
It might be cleaner to implement `String` on the `ClientCred` type (and then remember to wrap it in `logutils.StringerAttr`) or maybe even just implement `LogValue`.

Reply:
`LogValue` is pretty cool, TIL. It ends up being logged like this. Admittedly, `slog.GroupValue` seems to look better with JSON formatters but I'll take this.

> `2024-07-30T16:58:15+02:00 INFO [VNET:DAEM] Received VNet config socket_path:/var/folders/1x/xhqmbxsd3_v9zdv78ybr14zr0000gn/T/vnetffa74891-73de-43c8-9df1-23d80ca0873c.sock ipv6_prefix:fde3:5ebb:fece:: dns_addr:fde3:5ebb:fece::2 home_path:/Users/fav/.tsh creds_valid:true egid:20 euid:501 trace_id:8a25ef3f4bcc9bf975b59e63996dbe41 span_id:ff3cdafcd7d941c7 daemon/service_darwin.go:75`